backend/src/v2/compiler/argocompiler/argo.go

```diff
@@ -177,6 +183,7 @@ type workflowCompiler struct {
 	driverCommand   []string
 	launcherImage   string
 	launcherCommand []string
+	cacheEnabled    *bool
```

Comment:
I'm wondering if it'd be better call this `cacheDisabled` so the default of `false` is the current behavior. Then we wouldn't need to use pointers.

Reply:
Done.